rali/rali_pybind/example/tf_petsTrainingExample/train_withRALI_withTFRecordReader.py

```diff
@@ -0,0 +1,190 @@
+from amd.rali.plugin.tf import RALIIterator
+from amd.rali.pipeline import Pipeline
+import amd.rali.ops as ops
+import amd.rali.types as types
+import os
+import sys
+import tensorflow.compat.v1 as tf
+tf.disable_v2_behavior()
+
+import numpy as np
+import tensorflow_hub as hub
+
+
+
+
+
+############################### CHANGE THESE GLOBAL VARIABLES APPROPRIATELY ###############################
+
+RECORDS_DIR = './tfr/'
+NUM_CLASSES = 37
+LEARNING_RATE = 0.005
+NUM_TRAIN_STEPS = 2775
+TRAIN_BATCH_SIZE = 8
+EVAL_EVERY = 100
+DATASET_DOWNLOAD_AND_PREPROCESS = True
+RUN_ON_HOST = True
+
+############################### CHANGE THESE GLOBAL VARIABLES APPROPRIATELY ###############################
+
+
+
+
+
+######################################## NO CHANGES IN CODE NEEDED ########################################
+
+TRAIN_RECORDS_DIR = RECORDS_DIR + 'train/'
+VAL_RECORDS_DIR = RECORDS_DIR + 'val/'
+
+class HybridPipe(Pipeline):
+	def __init__(self, feature_key_map, tfrecordreader_type, batch_size, num_threads, device_id, data_dir, crop, rali_cpu = True):
+		super(HybridPipe, self).__init__(batch_size, num_threads, device_id, seed=12 + device_id,rali_cpu=rali_cpu)
+		self.input = ops.TFRecordReader(path=data_dir, index_path = "", reader_type=tfrecordreader_type, user_feature_key_map=feature_key_map, 
+			features={
+				'image/encoded':tf.FixedLenFeature((), tf.string, ""),
+				'image/class/label':tf.FixedLenFeature([1], tf.int64,  -1),
+				'image/filename':tf.FixedLenFeature((), tf.string, "")
+			}
+		)
+		rali_device = 'cpu' if rali_cpu else 'gpu'
+		decoder_device = 'cpu' if rali_cpu else 'mixed'
+		device_memory_padding = 211025920 if decoder_device == 'mixed' else 0
+		host_memory_padding = 140544512 if decoder_device == 'mixed' else 0
```

Comment:
Codacy found an issue: [local variable 'host_memory_padding' is assigned to but never used (F841)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)